### PR TITLE
Content update

### PR DESCRIPTION
**Pull request by Greg-Watanabe**

### CODE REVIEW

File: sample_site/_includes/navigation.njk [hunk 1]
```diff
@@ -6,26 +6,32 @@
...
+    </li>
     <li class="nav-item">
       <a
         class="first-level-link"
         href="{{(collections.all | allSamples)[0].url}}"
         >Samples</a
       >
     </li>
+    <li class="nav-item">
+      <a class="first-level-link" href="/top-task-1/">Top task 1</a>
```
Comment (carterm) [inline sample_site/_includes/navigation.njk:20]: Needs to be a file link "xxx.html" to support desktop viewing.

**Review by carterm — CHANGES_REQUESTED** [review]
Make sure it runs as a static from the _site folder directly too.  So far I only found one issue with the "Top task 1" link.  Great work!

File: sample_site/_includes/navigation.njk (hunk 1 above)
Comment (extinah) [reply sample_site/_includes/navigation.njk:20]: That was my comment, too. Same for task 2 overview link.

Comment (Greg-Watanabe) [reply sample_site/_includes/navigation.njk]: this is done

**Review by carterm — APPROVED** [review]
Good update to me

**Pull request merged**